addons/account/models/account_move_line.py

```diff
@@ -17,7 +17,7 @@ class AccountMoveLine(models.Model):
     _name = "account.move.line"
     _inherit = "analytic.mixin"
     _description = "Journal Item"
-    _order = "date desc, move_name desc, sequence, id"
+    _order = "date desc, move_name desc, id"
```

Comment:
but is not this one important on invoice lines?

Reply:
No, the order is explicitly `sequence` in the view. I don't think that it has a meaningful impact anywhere else.